### PR TITLE
NumFOCUS Sponsored Project blog post

### LINKED ISSUE #120: NumFOCUS blog post
Now that we are a [fiscally sponsored NumFOCUS project](https://numfocus.org/project/mdanalysis) we want to announce this (once NF releases their official blog post). 
- [ ] our blog post
- [ ] twitter announcement
- [ ] dev and user mailing list announcement

We need a draft blog post ready to go that briefly summarizes who we are, what we do, what it means to be a NumFOCUS sponsored project, and what good things might come out of it in the future.

### PR DESCRIPTION
**Pull request by orbeckst**
Fixes #120 

- overview
- link to governance/coredevs
- outlook
- donate button

### CODE REVIEW

**Comment by orbeckst** [comment]
@MDAnalysis/coredevs please change/review as necessary.

One thing is definitely problematic: It is not clear who the @MDAnalysis/coredevs really are. The link https://github.com/orgs/MDAnalysis/teams/coredevs/members is only visible if you're an MDAnalysis Org member. I have not found a way to make a team globally visible on GitHub. Suggestions on how we can be more transparent are welcome!!

**Comment by kain88-de** [comment]
We can keep a list of core devs in the government section. It means we have to places to maintain for the list. However it is easily visible.

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+sponsored project means:
+
+[**MDAnalysis**]({{ site.baseurl }}/about) is an open source software
+project that produces the *MDAnalysis Python library* for the analysis
+of computer simulations of many-body systems at the molecular scale,
+spanning use cases from interactions of drugs with proteins to novel
+materials. It is widely used in the scientific community and is
+written by scientists for scientists. The MDAnalysis Project is
+represented by the [MDAnalysis Core Developers]({{ site.baseurl
+}}/about#governance).
```
Comment (tylerjereddy) [inline _posts/2020-02-14-numfocus-sponsored-project.md]: As Max suggests, maybe a simple text list for "core devs" would be appropriate if you want something to link to.

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+[**NumFOCUS**](https://numfocus.org/) is an organization that promotes
+open practices in research, data, and scientific computing by serving
+as a fiscal sponsor for open source projects and organizing
+community-driven educational programs. In brief, they provide
+financial and logistic infrastructure to [many open source
+projects](https://numfocus.org/sponsored-projects) that fullfil
```
Comment (tylerjereddy) [inline _posts/2020-02-14-numfocus-sponsored-project.md]: `fulfil` of `fulfill`, I think, depending on country..

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+important roles in their communities. NumFOCUS is a 501(c)(3) public
+charity in the United States and it is able to accept
+donations for and on-behalf of projects and act as a grantee.
+
+NumFOCUS will accept funds on behalf of the MDAnalysis Project and
+then make it available to us. By becoming a [fiscally sponsored
```
Comment (tylerjereddy) [inline _posts/2020-02-14-numfocus-sponsored-project.md]: `it` -> `them` I think

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+project](https://numfocus.org/projects-overview), we agreed that **all
+software produced under the MDAnalysis umbrella is and will be free
+Open Source software** and that any funds will be spend compliant with
```
Comment (tylerjereddy) [inline _posts/2020-02-14-numfocus-sponsored-project.md]: `spend` -> `spent`, I think

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+So if you want to donate to MDAnalysis: you can now do this easily
+with the click of this button:
+
+{{ site.numfocus.donate_button }}
+
+More importantly, with NumFOCUS as a partner, MDAnalysis gains a
+long-term perspective for stable development that becomes less
+dependent on individuals and academic institutions. It allows the
+project obtain its own funding and move in directions that best
```
Comment (tylerjereddy) [inline _posts/2020-02-14-numfocus-sponsored-project.md]:
```suggestion
project to obtain its own funding and move in directions that best
```

File: _posts/2020-02-14-numfocus-sponsored-project.md (hunk 2 above)
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: changed

File: _posts/2020-02-14-numfocus-sponsored-project.md (hunk 3 above)
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: changed

File: _posts/2020-02-14-numfocus-sponsored-project.md (hunk 4 above)
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: fixed

File: _posts/2020-02-14-numfocus-sponsored-project.md (hunk 5 above)
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: fixed

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+NumFOCUS will accept funds on behalf of the MDAnalysis Project and
+then make it available to us. By becoming a [fiscally sponsored
+project](https://numfocus.org/projects-overview), we agreed that **all
+software produced under the MDAnalysis umbrella is and will be free
+Open Source software** and that any funds will be spend compliant with
+NumFOCUS' tax-exempt status: what this means in practice is that **all
+funds will go to further the growth and well-being of the project**.
+
+So if you want to donate to MDAnalysis: you can now do this easily
+with the click of this button:
```
[the remark this answers is not in the document]
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: changed

File: _posts/2020-02-14-numfocus-sponsored-project.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+More importantly, with NumFOCUS as a partner, MDAnalysis gains a
+long-term perspective for stable development that becomes less
+dependent on individuals and academic institutions. It allows the
+project obtain its own funding and move in directions that best
+benefit its community. For example, it will become easier to organize
+workshops, arrange for developers to travel, and to hire developers.
+
+For users and developers there really are not any changes to how the
+MDAnalysis Project operates: it remains a friendly and respectful
```
[the remark this answers is not in the document]
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: changed

File: _posts/2020-02-14-numfocus-sponsored-project.md (hunk 1 above)
Comment (orbeckst) [reply _posts/2020-02-14-numfocus-sponsored-project.md]: added to About

**Comment by orbeckst** [comment]
@MDAnalysis/coredevs we added an explicit list of everyone in https://github.com/orgs/MDAnalysis/teams/coredevs/members to the updated Governance Section of the website (file `about.md`) because the teams are not visible to the general public. Please have a quick look at the update (which comes together with our blog post for becoming a NumFOCUS sponsored project) and let me know if you have any comments/suggestions for changes.

**Comment by orbeckst** [comment]
NF is announcing today with blog post and in their newsletter so I’m merging.

**Pull request merged**